packages/application-generic/src/services/queues/standard-queue.service.ts

```diff
@@ -21,4 +22,18 @@ export class StandardQueueService extends QueueBaseService {
 
     this.createQueue();
   }
+
+  public async addMinimalJob(data: IStandardJobDto) {
+    return await super.addMinimalJob(data);
+  }
+
+  public async add(data: unknown) {
+    // in order to implement we need to know what should be the data dto first
+    throw new Error('Not implemented');
+  }
+
+  public async addBulk(data: unknown[]) {
+    // in order to implement we need to know what should be the data dto first
+    throw new Error('Not implemented');
+  }
```

Comment:
We do not use those methods at the moment, therefore i can not tell what should be the DTO here, i wonder if we should keep the exception or allow any here.

Reply:
The `add` function should be deprecated for the `standard queue` right?

Reply:
Are you saying we should delete it? In my view, we should get rid of addMinimalJob and all its related stuff like Worker.extractMinimalJobData. Instead,  just use the add function in the standard queue with the minimal data.

I didn't change it because I made this PR mainly to add DTO types in the code. So, the PR focuses on adding the necessary logic for these types without messing with the main logic.